apps/web/src/components/forms/advanced-monitor-form.tsx

```diff
@@ -65,6 +71,7 @@ export function AdvancedMonitorForm({ defaultValues, workspaceSlug }: Props) {
       method: defaultValues?.method ?? "GET",
     },
   });
+  const methodWatch = form.watch("method");
```

Comment:
Good thing on the rename! I'd suggest to go for `watchMethod` as this is our current naming convention.